website/docs/faqs/Core/install-python-compatibility.md

```diff
@@ -5,26 +5,14 @@ sidebar_label: 'Python version'
 id: install-python-compatibility
 ---
 
-<VersionBlock firstVersion="1.6">
+You can use the followign table to match your dbt-core version with the compatible version of Python. New dbt minor versions will add support for new Python3 minor versions as soon as all dependencies can support it. In addition, dbt minor versions will drop support for old Python3 minor versions right before they reach [end of life](https://endoflife.date/python).
 
-The latest version of `dbt-core` is compatible with Python versions 3.8, 3.9, 3.10 and 3.11.
-
-</VersionBlock>
-
-<VersionBlock firstVersion="1.4" lastVersion="1.5">
-
-The latest version of `dbt-core` is compatible with Python versions 3.7, 3.8, 3.9, 3.10 and 3.11.
-
-</VersionBlock>
-
-<VersionBlock lastVersion="1.3">
-
-The latest version of `dbt-core` is compatible with Python versions 3.7, 3.8, 3.9, and 3.10
-
-</VersionBlock>
+| dbt-core → <br/> Python ↓ | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 | 1.7 | 1.8 |
```

Comment:
@matthewshaver @mirnawong1 and @nghi-ly I wonder if we should switch tables with ✅ and ❌  to this greyed out x? It's so much easier to read. You focus on the ✔️ instead of everything

Reply:
@runleonarun : for me, the page feels a little bit "washed out" with the ✔️